images/cilium/Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG CILIUM_RUNTIME_IMAGE=quay.io/cilium/cilium-runtime:8fe001a11f25ad9e6676c19b0
 
 # cilium-envoy from github.com/cilium/proxy
 #
-FROM quay.io/cilium/cilium-envoy:b3bb275dcaf4a74ae956e52fa408b196c0d52152@sha256:f3c7645c0ff1d7552b79927023cf540f82b40263780644f75291ed0bb1965bee as cilium-envoy
+FROM quay.io/cilium/cilium-envoy-dev:59ac5af7d75273c2f2c0080be55eed4b07b4a1d8@sha256:48e6eb6024c84cc1114c7135bd42eb2e7797ec05d9366ae603a375a039f05188 as cilium-envoy
```

Comment:
a very basic question but what's the difference between quay.io/cilium/cilium-envoy and quay.io/cilium/cilium-envoy-dev?